src/main/scala/com/framex/core/FrameX.scala

```diff
@@ -1,53 +1,52 @@
 package com.framex.core
 
-import scala.collection.mutable
 import scala.collection.mutable.ListBuffer
 import scala.reflect.ClassTag
 import scala.reflect.runtime.{universe => ru}
 
-class FrameX(var data: Vector[Vector[ElemX]]) {
-
-  var columnMap: mutable.HashMap[String, Int] = mutable.HashMap()
-
-  def shape() : (Int, Int) = {
+class FrameX(var data: Vector[Vector[ElemX]], var columnMap: Map[String, Int] = Map()) {
 
+  def shape(): (Int, Int) = {
     (data(0).size, data.size)
   }
 
-  def ndim = 2
+  def head = data.map(_.head)
+
+  def tail() = data.map(_.tail)
```

Comment:
Is it use Immutable.Vector.tail() method here?

It seems that vector default tail method react differently with Pandas.tail, which return last 5 row data by default :)